src/main/java/io/kestra/plugin/amqp/AbstractAmqpConnection.java

```diff
@@ -47,14 +48,18 @@ void parseFromUrl(RunContext runContext, String url) throws IllegalVariableEvalu
         URI amqpUri = new URI(runContext.render(url));
 
         host = amqpUri.getHost();
```

Comment:
Please add
```
if (host != null) {
  throw new IllegalArgumentException("Can't provide both `url` and `host`")
}
```

to prevent side effects like "I forgot to remove url property so it reaches wrong server".